lib/src/logic.dart

```diff
@@ -575,6 +575,89 @@ class Logic {
   /// Greater-than-or-equal-to.
   Logic operator >=(dynamic other) => GreaterThanOrEqual(this, other).out;
 
+  /// Shorthand Increment.
+  /// By default for a [Logic] var, if no [incrVal] is provided result is ++var
+  /// else result is var+=[incrVal]
+  ///
+  /// ```dart
+  ///
+  /// Logic a = Logic(width: 8);
```

Comment:
In the example, you don't need to add inputs and outputs, the only part that's relevant is that they are `Logic` type